### PR TITLE
CB-13489 Mount and format scripts will handle no attached or local disks

### PR DESCRIPTION
**Pull request by daszabo**
See detailed description in the commit message.

### CODE REVIEW

File: orchestrator-salt/src/main/resources/salt/salt/disks/mount/scripts/find-device-and-format.j2 [hunk 1]
```diff
@@ -125,6 +125,10 @@ main() {
     ephemeral_device_name_list=$(get_ephemeral_device_names "$device_name_list")
     log $LOG_FILE found devices: $device_name_list $ephemeral_device_name_list
 
+    if [[ "$device_name_list" != "" && "$ephemeral_device_name_list" != "" ]]; then
+        exit_with_code $LOG_FILE $EXIT_CODE_OK "no devices to format"
+    fi
+
     format_disks_if_unformatted "$device_name_list $ephemeral_device_name_list"
```
Comment (lajosrodek) [inline orchestrator-salt/src/main/resources/salt/salt/disks/mount/scripts/find-device-and-format.j2:132]: So proceed if at least one variable is empty, and exit if both are nonempty? What is the rationale here?

Comment (lajosrodek) [reply orchestrator-salt/src/main/resources/salt/salt/disks/mount/scripts/find-device-and-format.j2:132]: Perhaps your rather meant to exit if both variables are empty?

Comment (daszabo) [reply orchestrator-salt/src/main/resources/salt/salt/disks/mount/scripts/find-device-and-format.j2]: Oops, good catch, thanks

**Review by lajosrodek — APPROVED** [review]
LGTM

**Review by nadamdb — APPROVED** [review]
LGTM

**Pull request merged**